lib/Twig/Template.php

```diff
@@ -361,13 +361,32 @@ protected function displayWithErrorHandling(array $context, array $blocks = arra
      */
     protected function getAttribute($object, $item, array $arguments = array(), $type = Twig_Template::ANY_CALL, $isDefinedTest = false, $ignoreStrictCheck = false)
     {
+        if (is_object($object)) {
+            $class = get_class($object);
+            if (!isset(self::$cache[$class])) {
+                self::$cache[$class] = $this->getCacheForClass($class);
```

Comment:
for the non-debug mode, what about keeping this cache on disk? Would that makes things faster? It looks like there is a lot going on in the `getCacheForClass()` method and I cannot believe that it does not have any impacts on real-world templates.